openshift/ci-operator/build-image/setup-logging-for-e2e.sh

```diff
@@ -6,28 +6,164 @@
+cat > elasticsearch-catalogsourceconfig.yaml << EOF
+apiVersion: "marketplace.redhat.com/v1alpha1"
+kind: "CatalogSourceConfig"
+metadata:
+  name: "elasticsearch"
+  namespace: "openshift-marketplace"
+spec:
+  targetNamespace: "openshift-operators"
+  packages: "elasticsearch-operator"
+EOF
+oc create -n openshift-marketplace -f elasticsearch-catalogsourceconfig.yaml
+oc get -n openshift-marketplace CatalogSourceConfig | grep elasticsearch || :
+
+# Create the subscription for elasticsearch in the namespace openshift-operators:
+cat > elasticsearch-subscription.yaml << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  generateName: elasticsearch-
+  namespace: openshift-operators
+spec:
+  source: elasticsearch
+  sourceNamespace: openshift-operators
+  name: elasticsearch-operator
+  startingCSV: elasticsearch-operator.v0.0.1
+  channel: preview
+EOF
+oc create -n openshift-operators -f elasticsearch-subscription.yaml
+oc get -n openshift-operators subscriptions | grep elaasticsearch || :
+
+# Create the CatalogSourceConfig for cluster-logging in the namespace openshift-marketplace:
+cat > cluster-logging-catalogsourceconfig.yaml << EOF
+apiVersion: "marketplace.redhat.com/v1alpha1"
+kind: "CatalogSourceConfig"
+metadata:
+  name: "cluster-logging"
+  namespace: "openshift-marketplace"
+spec:
+  targetNamespace: "openshift-logging"
+  packages: "cluster-logging"
+EOF
+oc create -n openshift-marketplace -f cluster-logging-catalogsourceconfig.yaml
+oc get -n openshift-marketplace CatalogSourceConfig | grep logging || :
+
+# create the subscription in the namespace openshift-logging:
+cat > cluster-logging-subscription.yaml << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  generateName: cluster-logging-
+  namespace: openshift-logging
+spec:
+  source: cluster-logging
+  sourceNamespace: openshift-logging
+  name: cluster-logging
+  startingCSV: clusterlogging.v0.0.1
+  channel: preview
+EOF
+oc create -n openshift-logging -f cluster-logging-subscription.yaml
+oc get -n openshift-logging subscriptions | grep logging || :
+
+# at this point, the cluster-logging-operator should be deployed in the
+# openshift-logging namespace
+oc project openshift-logging
+
+wait_func() {
+    oc -n openshift-operators get pods 2> /dev/null | grep -q 'elasticsearch-operator.*Running' && \
+    oc get pods 2> /dev/null | grep -q 'cluster-logging-operator.*Running'
+}
+if ! wait_for_condition wait_func 300 > ${ARTIFACT_DIR}/test_output 2>&1 ; then
+    echo ERROR: one of or both of elasticsearch-operator and cluster-logging-operator pod not running
+    logging_err_exit
 fi
+
+# get the OLM pod
+olmpod=$( oc -n openshift-operator-lifecycle-manager get pods | awk '/^olm-operator-.* Running / {print $1}' )
+
+# disable the OLM so that we can change images in the cluster-logging-operator
+oc -n openshift-operator-lifecycle-manager scale --replicas=0 deploy/olm-operator
```

Comment:
Why do we need to do this?  Is it because the `ci-operator` is not testing the `cluster-logging-operator` associated with the PR?

Reply:
Yes.   This test does not use ci-operator.